sci-libs/hdf5/hdf5-1.10.1.recipe

```diff
@@ -6,7 +6,7 @@ HOMEPAGE="http://www.hdfgroup.org/HDF5/"
 COPYRIGHT="2011-2016 The HDF Group,
 	The Board of Trustees of the University of Illinois."
 LICENSE="BSD (3-clause)"
-REVISION="1"
+REVISION="2"
```

Comment:
We don't need to bump `REVISION`, unless the builders are doing things based on the result of `TEST()`.

Reply:
Ah, will revert :)